cluster/gce/util.sh

```diff
@@ -762,9 +762,6 @@ function construct-linux-kubelet-flags {
   flags+=" --experimental-check-node-capabilities-before-mount=true"
   # Keep in sync with the mkdir command in configure-helper.sh (until the TODO is resolved)
   flags+=" --cert-dir=/var/lib/kubelet/pki/"
-  # Configure the directory that the Kubelet should use to store dynamic config checkpoints
-  flags+=" --dynamic-config-dir=/var/lib/kubelet/dynamic-config"
```

Comment:
Double check that this doesn't break the node e2e tests.

Reply:
Well, I guess they passed, so ok then :D.

Given the whole `cluster/` directory is deprecated, I'm not sure we offer any guarantees for these scripts, but I do wonder if anyone might be depending on this who would be broken, and who might appreciate some forewarning.

Reply:
e2e tests will also set directory when the feature flag is set. Which is safer.

I understand the risk here. But since it only suppose to be used in e2e tests, I'd suggest to enforce it now. Reverting will be straightforward

Reply:
Ok, SGTM